packages/unplugin-typia/tests/fixtures/__snapshots__/vite/random.js

```
@@ -39,8 +1,120 @@
+var hasRequired_randomInteger;
+function require_randomInteger() {
+  if (hasRequired_randomInteger) return _randomInteger;
+  hasRequired_randomInteger = 1;
+  Object.defineProperty(_randomInteger, "__esModule", { value: true });
+  _randomInteger._randomInteger = void 0;
+  const _randomInteger$1 = (props) => {
+    var _a, _b, _c, _d;
+    let minimum = (_a = props.minimum) !== null && _a !== void 0 ? _a : ((_b = props.multipleOf) !== null && _b !== void 0 ? _b : 1) * (props.maximum === void 0 ? 0 : props.maximum - 100);
+    if (props.minimum !== void 0 && props.exclusiveMinimum === true)
+      minimum++;
+    let maximum = (_c = props.maximum) !== null && _c !== void 0 ? _c : ((_d = props.multipleOf) !== null && _d !== void 0 ? _d : 1) * (props.minimum === void 0 ? 100 : props.minimum + 100);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)